config/initializers/extensions.rb

```diff
@@ -0,0 +1 @@
+require "active_record_extension"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.